GetDisplayUserInfo/GetDisplayUserInfo/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace GetDisplayUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("What is your first name?");
+            string firstName = Console.ReadLine();
+
+            Console.WriteLine("Hello, " + firstName + ". " + "It's very nice to meet you!");
+
+            Console.WriteLine("What is your last name?");
+            string lastName = Console.ReadLine();
+
+            Console.WriteLine(lastName + "... " + "that is a very unique last name!");
+
+            Console.WriteLine("What is your favorite number?");
+            string favoriteNumberResponse = Console.ReadLine();
+            int favoriteNumber = int.Parse(favoriteNumberResponse);
```

Comment:
This is good and works, and I'm always a fan of starting with something that's working/simple.  Could this now be improved so that it doesn't throw an exception if the user enters something that's not an integer?